### PR TITLE
adds user-ids-by-hostnames handler

### PR DESCRIPTION
**Pull request by ddellacosta**
https://www.pivotaltracker.com/story/show/128103513

Also tested manually using docker/rabbitmq

### CODE REVIEW

**Comment by ddellacosta** [comment]
So, I just realized something, which is that I'm not returning these in a vector, which is a requirement of the card.  It's trivial to do this, but does it really matter?  The only thing that cares about the ordering should be the user of the return value of this endpoint, and it can order it however it wants.

cc @tank157 @danielglauser

File: src/user_works/queue.clj [hunk 1]
```diff
@@ -48,14 +48,21 @@
...
                                                channels/export-requests
                                                channels/export-responses)
 
+          by-hostnames-ch (wire-up/incoming-service
+                        connection
+                        "user-works.user.user-ids-by-hostnames"
+                        (config [:rabbitmq :queues "user-works.user.user-ids-by-hostnames"])
+                        channels/by-hostnames-requests
+                        channels/by-hostnames-responses)
+
```
Comment (cap10morgan) [inline src/user_works/queue.clj]: Can you fix the indentation here?

**Comment by cap10morgan** [comment]
Small indentation fix needed. After that, LGTM.

**Comment by tank157** [comment]
@ddellacosta I concur, it's not so important that this component return an ordered sequence, the handler in `turbovote-admin-works` that uses these ID's can order them as it sees fit.

**Pull request merged**